### ISSUE TITLE
[EN][QUEST][The Storm Peaks]The Lonesome Watcher

### ISSUE THREAD

**Issue by Snake90** [Atlantiss/BugTracker#10264]
[/]: # (** DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED **)

Problem with the quest [The Lonesome Watcher] : 
https://www.wowhead.com/quest=12877/the-lonesome-watcher

**Description:** Impossible to end the quest because every interaction with the NPC is impossible. 

**Current behaviour:** Nothing because you CAN'T talk to the NPC

**Expected behaviour:** After you dropped the Frostgrip's Signet Ring, you have to go to Creteus to end the quest but it's impossible to talk to him. 
 
** Proof : ** ![wowscrnshot_122618_221110](https://user-images.githubusercontent.com/17967508/50457435-c5fee100-095b-11e9-9ba1-8419d47465ad.jpg)

**Steps to reproduce the problem:**

1. Speak to Yorg Stormheart in FrostHold and take the quest [An Experienced Guide] then go to Drom Frostgrip to end the quest and take [The Lonesome Watcher]
2. Kill an Iron Dwarf Monitor and drop the Frostgrip's Signet Ring
3. Go to Creteus and try to speak to him ... You can't. 

**Revision:** Current server revision (You can see current server revision by typing ".server info" in the in-game chat.)

**Comment by TranceAccount** [comment]
This is probably a phasing problem, and in [this video](https://www.youtube.com/watch?v=QED8hmP51gU) at the ending you see Creteus wandering around. You can probably find that Createus spawn in order to complete your quest for now.

**Comment by Snake90** [comment]
No, I can't find him. There is a Creteus but these one isn't in the good phase with me so I can't end the quest ... Normally, at this point, he's walking around the area but in game he's standing with another Creteus who is dead that isn't normal. And he is in phase with the quest [Fury of the Frostborn King].
